packages/cactus-plugin-keychain-aws-sm/README.md

```diff
@@ -0,0 +1,3 @@
+# `@hyperledger/cactus-plugin-keychain-aws-sm`
+
+## TO-DO
```

Comment:
@jagpreetsinghsasan Please create a follow-up issue to write a README.md file with examples for the package.

Reply:
Created #968